commands/init.go

```diff
@@ -45,13 +45,14 @@ func transformInitArgs(args *Args) error {
 		err    error
 	)
 
-	if args.IsParamsEmpty() {
+	words := args.Words()
```

Comment:
How can we distinguish between a "word" that stands on its own and words the are values for options such as `--template`, `--separate-git-dir`, `--shared`?